packages/shared/src/hooks/useOnboardingAnimation.ts

```diff
@@ -0,0 +1,40 @@
+import { useCallback, useState } from 'react';
+import type Router from 'next/router';
+import { useRouter } from 'next/router';
+import useDebounce from './useDebounce';
+
+interface UseOnboardingAnimation {
+  isAnimating: boolean;
+  postOnboardingRedirect: (param: Parameters<typeof Router.replace>[0]) => void;
+  finishedOnboarding: boolean;
+  onFinishedOnboarding: () => void;
+}
+
+interface UseOnboardingAnimationProps {
+  animationMs?: number;
+}
+
+const DEFAULT_ANIMATION_MS = 2750;
+
+export const useOnboardingAnimation = ({
+  animationMs = DEFAULT_ANIMATION_MS,
+}: UseOnboardingAnimationProps = {}): UseOnboardingAnimation => {
```

Comment:
Recovered from onboarding fake loader experiment as requested in RFC/spec.

Reply:
Non blocking buy maybe we should rename it a bit to avoid `onboarding` might become confusing down the line.

Reply:
9bebc5273d894134e7421280e9a0745de0e4de85